mvp-spec/audit-log-design.md

```diff
@@ -135,14 +98,56 @@ the Contracting Party handles those objects.
 }
 ```
 <!--partial-end-->
+</details>
+
+
+## Reference the Addressable Content
+
+To be able to match an Audit Log to an Addressable Content when there are many in the webpage, the DOM tag that contains the Addressable Content must have the attribute `paf-transaction` that contains the Transaction Id.
 
-# Display of the Audit UI
+```html
+<div paf-transaction="ad_container">
+    <div>This is an ad.</div>
+    <a  class="prebid_sso_audit_button">Audit Log</a>
+</div>
+```
+
+## Store the Audit Log
 
-## Web Page interface
+### Alternative 1 - HTML
+
+Each Addressable Content has an Audit Log in a webpage. The Publisher stores the Audit Log on the page via a `<meta>` tag.
```

Comment:
I'm not sure in which webpage the meta tag will be used.
If it's in the "parent" webpage, then there would be multiple `meta` tags (one per transaction), correct?
In this case, I would recommend this phrasing:
> The Publisher stores the Audit Log on the page via `<meta>` tags, one per transaction.